bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Election.cs

```diff
@@ -112,29 +113,74 @@ public unsafe ElementModQ CryptoExtendedBaseHash
             }
         }
 
+        /// <summary>
+        /// Get a linked list containing the extended data of the election.
+        /// </summary>
+        public unsafe LinkedList ExtendedData
+        {
+            get
+            {
+                var status = NativeInterface.CiphertextElectionContext.GetExtendedData(
+                    Handle, out NativeLinkedList value);
+                if (status != Status.ELECTIONGUARD_STATUS_SUCCESS)
+                {
+                    Console.WriteLine($"GetExtendedData Error Status: {status}");
+                    return null;
```

Comment:
returning null in the property accessor rather than throwing, though i could be convinced this pattern is incorrect since if the code comes back as a non-success it likely means it is irrecoverable.

Reply:
I would argue that this is complicated enough that it should be a method not a property getter, and that then throwing would make more sense.  But we talked about this, and this approach is fine for now because iirc you're primary objective is to match the underlying native pattern and the method approach would be more code.